build.Dockerfile

```diff
@@ -1,11 +1,8 @@
-FROM golang:1.8
+FROM golang:1.9-alpine
```

Comment:
OK, missed that it's actually rc2 already.

I'm still not sure it's a good idea to merge this before Go 1.9 final is actually released. I have seen things break in subtle ways over at least the last two Go releases. I'd prefer to take the time after the code freeze to iron out potential problems.